sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -572,12 +572,18 @@ def _add_argparse_args(cls, parser):
 
 
 class CrossLanguageOptions(PipelineOptions):
+  @staticmethod
+  def _beam_services_from_enviroment():
+    return json.loads(os.environ.get('BEAM_SERVICE_OVERRIDES') or '{}')
+
   @classmethod
   def _add_argparse_args(cls, parser):
     parser.add_argument(
         '--beam_services',
-        type=json.loads,
-        default={},
+        type=lambda s: {
+            **cls._beam_services_from_enviroment(), **json.loads(s)
```

Comment:
LGTM, thanks. PTAL at doc string while we are at it.
- do we still want to mention gradle details there?
- do we want to document reference BEAM_SERVICE_OVERRIDES there?
- any other changes you'd like to mention.